### PR TITLE
move status out of log package

### PR DESCRIPTION
**Pull request by BenTheElder**
breaking out a very small bit of the logging work, clearing things out of the exported log package that don't particularly belong there before adding the shiny new logging ...

### CODE REVIEW

File: cmd/kind/kind.go [hunk 1]
```diff
@@ -94,17 +94,14 @@ func Run() error {
 
 // Main wraps Run and sets the log formatter
 func Main() {
+	// TODO(bentheelder): replace this with a shimmable logger
 	// let's explicitly set stdout
-	log.SetOutput(os.Stdout)
+	log.SetOutput(os.Stderr)
```
Comment (neolit123) [inline cmd/kind/kind.go]: possibly, modify the above comment `set stdout` or keep as is?

Comment (BenTheElder) [reply cmd/kind/kind.go]: that should say stderr for sure, using stdout was a bug but not related to the rest of this PR ...

Comment (BenTheElder) [reply cmd/kind/kind.go]: fixed

**Pull request merged**